src/wlapplication_messages.cc

```diff
@@ -19,178 +19,207 @@
+	print_help_option(
+	   _("--rolling_autosave=[...]"), _("Use this many files for rolling autosaves."));
+	print_help_option(
+	   _("--metaserver=[...]"), _("Connect to a different metaserver for internet gaming."));
+	print_help_option(
+	   _("--metaserverport=[...]"), _("Port number of the metaserver for internet gaming."));
+	print_help_option(
+	   _("--addon_server=[...]"),
+	   _("Connect to a different github repository and branch from the add-ons manager."));
+	print_help_option(_("--nosound"), _("Starts the game with sound disabled."));
+	print_help_option(
+	   _("--fail-on-lua-error"), _("Force Widelands to crash when a Lua error occurs."));
+	print_help_option(_("--nozip"), _("Do not save files as binary zip archives."));
+	print_help_option(_("--editor"), _("Directly starts the Widelands editor."));
+	print_help_option(_("--editor=FILENAME"),
+	                  _("Directly starts the Widelands editor and loads the map FILENAME."));
+	print_help_option(
+	   _("--ai_training"),
+	   _("Enables AI training mode. See https://www.widelands.org/wiki/Ai%20Training/ for a full "
+	     "description of the AI training logic."));
+	print_help_option(
+	   _("--auto_speed"),
+	   _("In multiplayer games only, this will keep adjusting the game speed automatically, "
+	     "depending on FPS. Useful in conjunction with --ai_training."));
+	print_help_option(_("--new_game_from_template=FILENAME"),
+	                  _("Directly create a new singleplayer game configured in the given file. An "
+	                    "example can be found in `data/templates/new_game_template`."));
+	print_help_option(
+	   _("--scenario=FILENAME"), _("Directly starts the map FILENAME as scenario map."));
+	print_help_option(_("--loadgame=FILENAME"), _("Directly loads the savegame FILENAME."));
+	print_help_option(_("--replay=FILENAME"), _("Directly loads the replay FILENAME."));
+	print_help_option(
+	   _("--script=FILENAME"), _("Run the given Lua script after initialization. Only valid with "
+	                             "--scenario, --loadgame, or --editor."));
+	print_help_option(
+	   /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but it HAS TO BE
+	      CONSISTENT with the translation in the widelands textdomain */
+	   _("--auto_roadbuild_mode=[true|false]"),
+	   _("Whether to enter roadbuilding mode automatically after placing a flag that is not "
+	     "connected to a road."));
+
+	std::cout << std::endl << _("Graphic options:") << std::endl << std::endl;
+	print_help_option(
+	   /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but it HAS TO BE
+	      CONSISTENT with the translation in the widelands textdomain */
+	   _("--fullscreen=[true|false]"), _("Whether to use the whole display for the game screen."));
+	print_help_option(_("--xres=[...]"), _("Width of the window in pixels."));
+	print_help_option(_("--yres=[...]"), _("Height of the window in pixels."));
```

Comment:
`--x/y_res=n` "Width/height of the window in pixels. Don't use this with `--fullscreen=true`